source/settings/core.rst

```diff
@@ -3498,10 +3498,27 @@ See :ref:`settings` for list of all setting groups.
 .. dovecot_core:setting:: submission_ssl
    :default: no
    :todo: Indicate submission setting
-   :values: @boolean
+   :values: no, smtps, submissions, starttls
+
+   If enabled, SSL/TLS is used for the connection to :dovecot_core:ref:`submission_host`.
+
+   Available values:
+
+   ``no``
+
+      No SSL connection is used.
 
-   If enabled, use SSL/TLS to connect to :dovecot_core:ref:`submission_host`.
+   ``smtps``
+
+      An SMTPS connection (immediate SSL) is used.
+
+   ``submissions``
```

Comment:
i dont see this in code.

Reply:
can you also change fix: to settings: prefix, please

Reply:
> i dont see this in code.

I didn't go deeper that the config struct myself, which shows it as an allowed option: https://github.com/dovecot/core/blob/93eb0663336b17f66a67324de43e64bf0bc796c1/src/lib-smtp/smtp-submit-settings.c#L35

Possibly needs to be removed from there too?

Reply:
@sirainen @stephanbosch is this supposed to have submissions as option?

Reply:
I've removed _submissions_ as my code review also suggests this isn't implemented, hoping this unblocks PR whilst config struct can be updated independently.

Reply:
Looks like smtps and submissions are exactly the same:

```
		if (strcasecmp(set->submission_ssl, "smtps") == 0 ||
			strcasecmp(set->submission_ssl, "submissions") == 0)
			ssl_mode = SMTP_CLIENT_SSL_MODE_IMMEDIATE;
```

So it could be mentioned here, but I'd still call it SMTPS rather than SUBMISSIONS and say it's the same as smtps value.